blocks/basic/test/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
 add_ut_test(qa_Selector)
 add_ut_test(qa_sources)
 add_ut_test(qa_DataSink)
+
```

Comment:
Maybe we should use the described workaround in the issue:

> Execute calling boost::ut::cfg<boost::ut::override>.run(); from main and the test passes, but arguments such --help do nothing.

This would also help with the UT command line options.